### PR TITLE
Feature#45 styles student signup

### LINKED ISSUE #45: Add styles for Students sign up
# REQUIREMENTS
Add the styles for the student sign up view.
Reload the form a little on the right and put a brief description of the role on the left side

# Acceptance Criteria
- [ ] styles student sing up

### PR DESCRIPTION
**Pull request by RobertoCarMed**
# Description
- Fixed main page
- Styles for mobile layout added
- Desk styles reused to make student and teacher registration similar in desk layout

  ## Screenshots 
image
image
image

resolve #45

### CODE REVIEW

File: app/views/students/registrations/new.html.erb [hunk 1]
```diff
@@ -1,39 +1,73 @@
...
-  </div>
+    <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+      <%= render "students/shared/error_messages", resource: resource %>
 
-  <div class="field">
-    <%= f.label :name %><br />
-    <%= f.text_field :name, autofocus: true, autocomplete: "name" %>
-  </div>
+      <div class="field form-group">
+        <%= f.label :email %><br />
```
Comment (xFerchoVJ) [inline app/views/students/registrations/new.html.erb:40]: Bootstrap provide a class called [form-label](https://getbootstrap.com/docs/4.0/components/forms/) you can use it so you don't have to put that tag `</br>`

**Review by xFerchoVJ — APPROVED** [review]
Nice job!

**Review by SampabloC — APPROVED** [review]

**Review by brayandiazc — APPROVED** [review]
Good style

File: app/views/students/registrations/new.html.erb (hunk 1 above)
Comment (RobertoCarMed) [reply app/views/students/registrations/new.html.erb:40]: Thank you for information!

**Pull request merged**